webcompat/form.py

```diff
@@ -118,9 +119,24 @@ def get_labels(browser_name):
 def normalize_url(url):
     '''normalize URL for consistency.'''
     url = url.strip()
-    if not url.startswith(SCHEMES):
+    parsed = urlparse.urlparse(url)
+
+    if url.startswith(BAD_SCHEMES):
+        # if url starts with a bad scheme, parsed.netloc will be empty,
+        # so we use parsed.path instead
+        path = parsed.path.lstrip('/')
+        url = '%s://%s' % (parsed.scheme, path)
+        if parsed.query:
+            url += '?' + parsed.query
+        if parsed.fragment:
```

Comment:
Hmmmmmmm... I'm trying to think here. This should only matter in the case where the `if domain` part doesn't succeed (which if it did, we only display a domain). 

I think you're right, let's leave this as you've originally written it. Sorry!